CONTRIBUTING.md

```diff
@@ -17,6 +17,9 @@ Install dependencies:
 # optionally include [jwt] for the full test suite, but requires cryptography,
 #  and its dependent binary libraries
 pip install -e .[dev]
+
+# Or for zsh users
+pip install -e .\[dev\]
```

Comment:
Hmm, I don't recall ever needing this. Maybe something else is causing this?

Reply:
square brackets have special meaning in zsh most of the time you need to escape them if you want the literal